app/services/global_autocomplete.rb

```diff
@@ -29,97 +29,98 @@ def global_super_search_result
     results.uniq { |result| [result[:model_name], result[:id]] }
   end
 
-  def autocomplete_result(scope)
-    return [] unless scope && Pundit.policy!(user, GlobalAutocomplete.check_policy_klass(user, klass)).index?
-
-    self.current_scope = scope
-    order = scope[:query_order] || 'created_at DESC'
-    query = Pundit.policy_scope!(user, GlobalAutocomplete.check_policy_klass(user, klass))
-    query = query.joins(joins) if joins
-    query = query.where(where_query, search: "%#{params[:term]}%").order(order)
-
-    query = query.limit(50)
-    search_label = scope[:search_label] || :to_s
-
-    query.map do |record|
-      { columns: get_columns_values(columns, methods, record),
-        model_name: klass.name,
-        human_name: klass.model_name.human,
-        id: record.id,
-        label: record.send(search_label),
-        value: record.to_s,
-        scope_description: scope[:description] }
+  def self.check_policy_klass(user, klass)
+    if user.portal?
+      [RadCommon.portal_namespace, klass]
+    else
+      klass
     end
   end
 
-  def get_columns_values(columns, methods, record)
-    column_values = []
+  private
 
-    if columns.present?
-      columns.each do |column_name|
-        column_values.push(format_column_value(record[column_name]))
+    def autocomplete_result(scope)
+      return [] unless scope && Pundit.policy!(user, GlobalAutocomplete.check_policy_klass(user, klass)).index?
+
+      self.current_scope = scope
+      order = scope[:query_order] || 'created_at DESC'
+      query = Pundit.policy_scope!(user, GlobalAutocomplete.check_policy_klass(user, klass))
+      query = query.joins(joins) if joins
+      query = query.where(where_query, search: "%#{params[:term]}%").order(order)
+      query = query.where.not(id: params[:excluded_ids]) if params[:excluded_ids].present?
```

Comment:
this will fail if there are joins or includes in the query but will document that tech debt for later